### PR TITLE
Sites from google maps 482

### PR DESCRIPTION
**Pull request by ch1eroe1**
## Description

This code is tied to issue 482.  This allows a new site to be added into bety from the 02-modelsite.php page.

It is possible to add some regex field controls for the site addition modal.  I just need to understand what type of parameters we want to see (e.g. required fields, geometry controls that match psql constraints, etc.)
## Motivation and Context

Issue 482.
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue) 
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
## Checklist:

- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: web/02-modelsite.php [hunk 1]
```diff
@@ -80,13 +80,154 @@
...
+	#container { display: table; }
+	#row { display: table-row; }
+	#left, #right, #middle { display: table-cell; padding-left: 10px; }
+</style>
 <script type="text/javascript" src="jquery-1.7.2.min.js"></script>
 <?php if (!$offline) {?>
 <script type="text/javascript" src="//www.google.com/jsapi"></script>
 <?php }?>
+<?php if (!$offline) {?>
+<script type="text/javascript" src="//code.jquery.com/jquery-1.10.2.js"></script>
```
Comment (robkooper) [inline web/02-modelsite.php]: just update the jquery above, no need to load 2 copies of jquery

File: web/02-modelsite.php [hunk 2]
```diff
@@ -80,13 +80,154 @@
...
+<?php }?>
+<?php if (!$offline) {?>
```
Comment (robkooper) [inline web/02-modelsite.php]: use one if block not for every script

File: web/02-modelsite.php [hunk 3]
```diff
@@ -80,13 +80,154 @@
...
+<script type="text/javascript" src="//code.jquery.com/ui/1.11.4/jquery-ui.js"></script>
+<?php }?>
+<?php if (!$offline) {?>
+<link rel="stylesheet" type="text/css" href="//code.jquery.com/ui/1.11.4/themes/smoothness/jquery-ui.css"/>
+<?php }?>
 <script type="text/javascript">
+
+  //$(function()  {
+  $(window).load(function() {
```
Comment (robkooper) [inline web/02-modelsite.php]: can you clean up the following block to be more consistent with the next sections. In general I use 2 spaces for indentation. Also just 1 empty line between functions and remove some additional blank lines.

File: web/02-modelsite.php [hunk 4]
```diff
@@ -80,13 +80,154 @@
...
+      				jdata.find("marker").each(function() {
+        			var marker = jQuery(this);
+        			if (marker.attr("lat") == "" || marker.attr("lon") == "") {
+          				//console.log("Bad marker (siteid=" + marker.attr("siteid") + " site=" + marker.attr("sitename") + " lat=" + marker.attr("lat") + " lon=" + marker.attr("lon") + ")");
+        			} else {
+          				showSite(marker, marker.attr("siteid"));
+        			}
+      				});
+			});
+		// Prevent default posting of form
```
Comment (robkooper) [inline web/02-modelsite.php]: make sure to be consistent with your indentation.

**Comment by mdietze** [comment]
@ch1eroe1 it's been 1 month since @robkooper requested changes to this PR. The requested changes all look pretty minor. We have a release coming up next week and it would be really nice to get this in before then. Any chance you could devote a few minutes to clean up this week?

**Comment by robkooper** [comment]
I spoke with @ch1eroe1 and he said he would get this done today.

**Pull request merged**